test/functional/terminal/scrollback_spec.lua

```diff
@@ -441,11 +441,18 @@ describe("'scrollback' option", function()
     eq(-1, curbufmeths.get_option('scrollback'))
   end)
 
-  it('error if set on a normal buffer', function()
+  it('error if setlocal on a normal buffer', function()
     command('new')
-    execute('set scrollback=42')
+    execute('setlocal scrollback=42')
     feed('<CR>')
     eq('E474:', string.match(eval("v:errmsg"), "E%d*:"))
   end)
 
+  it('remains unchanged on a normal buffer', function()
+    command('new')
+    execute('set scrollback=42')
+    feed('<CR>')
+    eq(-1, curbufmeths.get_option('scrollback'))
```

Comment:
Will do. Seems like the `feed('<CR>')` can also be removed or is there some indeterminism to leave it in for?